### ISSUE TITLE
New UserNotice Found: rewardgift

### ISSUE THREAD

**Issue by crohde7707** [d-fischer/twitch#71]
## Feature Request

`⚠️  twitch-chat WARNING Unrecognized usernotice ID: rewardgift`

pretty sure has to do with the free winter emotes that the community gets when someone subs, but i don't have much data past that

**Comment by d-fischer** [comment]
I don't think it provides much value to be able to react to this programmatically. I could add some code to suppress the message for this particular ID, but as this is seasonal I'd say just ignore it.

**Comment by d-fischer** [comment]
Reopened since this seems to be a regular thing now.
